people.md

```diff
@@ -0,0 +1,60 @@
+# People in the governance structure
+
+This document lists the current people and institutions involved in
+the governance.
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
```

Comment:
For now I would alphabetize this list. Eventually, we may want to order by how long people have served.